### PR TITLE
Fix wrap on proofs (and other fixes)

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: desktop/renderer/style.css [hunk 1]
```diff
@@ -130,6 +130,9 @@ body {
 .hover-underline:hover {
   text-decoration: underline;
 }
+.hover-underline-container:hover .underline{
```
Comment (chrisnojima) [inline desktop/renderer/style.css]: if you want hover, but not the entire thing underlined, just some child

File: shared/more/components-tracker.render.desktop.js [hunk 2]
```diff
@@ -18,6 +18,11 @@ const proofGithub = proofGithubMaker('')
 
 const proofTwitter: Proof = {name: 'twitteruser', type: 'twitter', id: 'twitterId', state: normal, meta: metaNone, humanUrl: 'twitter.com', profileUrl: 'http://twitter.com'}
 const proofWeb: Proof = {name: 'thelongestdomainnameintheworldandthensomeandthensomemoreandmore.com', type: 'web', id: 'webId', state: normal, meta: metaNone, humanUrl: 'thelongestdomainnameintheworldandthensomeandthensomemoreandmore.com', profileUrl: ''}
+const proofWeb1: Proof = {name: 'thelongestdomainnameintheworldandthensomeandthensomemoreandmor.com', type: 'web', id: 'webId1', state: normal, meta: metaNone, humanUrl: 'thelongestdomainnameintheworldandthensomeandthensomemoreandmore.com', profileUrl: ''}
```
Comment (chrisnojima) [inline shared/more/components-tracker.render.desktop.js]: so we can see how wrap works better

File: shared/tracker/proofs.render.desktop.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const shell = electron.shell || electron.remote.shell
 
 import type {Proof, ProofsProps} from './proofs.render'
 
-export class ProofsRender extends Component {
+class ProofsRender extends Component {
```
Comment (chrisnojima) [inline shared/tracker/proofs.render.desktop.js]: lets move exports to the bottom always

File: shared/tracker/proofs.render.desktop.js [hunk 4]
```diff
@@ -104,7 +104,7 @@ export class ProofsRender extends Component {
     }
   }
 
-  renderProofRow (styles: Object, proof: Proof) {
+  renderProofRow (proof: Proof) {
```
Comment (chrisnojima) [inline shared/tracker/proofs.render.desktop.js]: don't need this passed in

File: shared/tracker/proofs.render.desktop.js [hunk 5]
```diff
@@ -112,106 +112,117 @@ export class ProofsRender extends Component {
     // TODO: State is deprecated, will refactor after nuking v1
     let isChecking = (proof.state === proofChecking)
 
+    const proofStyle = {
```
Comment (chrisnojima) [inline shared/tracker/proofs.render.desktop.js]: cleaner to figure this out outside the markup

File: shared/tracker/proofs.render.desktop.js [hunk 6]
```diff
@@ -112,106 +112,117 @@ export class ProofsRender extends Component {
...
-          </div>
-        </div>
-        {isChecking &&
-          <ProgressIndicator style={styles.loader} />
-        }
-        {!isChecking && proofStatusIcon &&
-          <Icon type={proofStatusIcon} style={styles.statusIcon} onClick={() => this.onClickProof(proof)} />
-        }
-      </div>
+      <p style={styleRow} key={proof.id}>
```
Comment (chrisnojima) [inline shared/tracker/proofs.render.desktop.js]: got rid of a lot of unnecessary nesting of divs

File: shared/tracker/proofs.render.desktop.js [hunk 7]
```diff
@@ -112,106 +112,117 @@ export class ProofsRender extends Component {
...
-      <div style={styles.container}>
-        {this.props.proofs.map(p => this.renderProofRow(styles, p))}
+      <div style={styleContainer}>
+        {this.props.proofs.map(p => this.renderProofRow(p))}
       </div>
     )
   }
 }
 
-const styles = {
```
Comment (chrisnojima) [inline shared/tracker/proofs.render.desktop.js]: not using the nested styles so we can eslint check this better

File: shared/tracker/render.desktop.js [hunk 8]
```diff
@@ -5,7 +5,7 @@ import React, {Component} from 'react'
 import Header from './header.render.desktop'
 import Action, {calcFooterHeight} from './action.render.desktop'
 import Bio from './bio.render.desktop'
-import {ProofsRender} from './proofs.render.desktop'
+import ProofsRender from './proofs.render.desktop'
```
Comment (chrisnojima) [inline shared/tracker/render.desktop.js]: why is this different?

**Comment by chrisnojima** [comment]
@keybase/react-hackers this fixes the wordwrap being broken in master and also the none metatag showing. refactored it so its smaller and simpler i think

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/tracker/proofs.render.desktop.js [hunk 9]
```diff
@@ -112,106 +112,117 @@ export class ProofsRender extends Component {
...
-          <Icon type={proofStatusIcon} style={styles.statusIcon} onClick={() => this.onClickProof(proof)} />
-        }
-      </div>
+      <p style={styleRow} key={proof.id}>
+        <Icon style={styleService} type={this.iconNameForProof(proof)} title={proof.type} onClick={onClickProfile} />
+        <span style={styleProofNameSection}>
+          <span style={styleProofNameLabelContainer}>
+            <Text inline className='hover-underline-container' type='Body' onClick={onClickProfile} style={proofStyle}>
+              <Text inline type='Body' className='underline' style={proofNameStyle}>{proof.name}</Text>
+              <Text className='no-underline' inline type='Body' style={styleProofType}><wbr>@{proof.type}</wbr></Text>
```
Comment (MarcoPolo) [inline shared/tracker/proofs.render.desktop.js]: Do you need this className?

**Comment by MarcoPolo** [comment]
:+1:

**Pull request merged**